### DISCUSSION TITLE
List of example scores and code created with neoscore

### DISCUSSION THREAD

**Discussion by jgarte** [DigiScore/neoscore#38] [Q&A] [answered]
Hi,

Is there a list of current scores and respective source code created with neoscore that I can peruse? I'm just curious if this exists yet.

**Comment by ajyoon** [comment, marked as answer]
Hey, and welcome! Neoscore is a very new library just released last month, so we don't have any fully written example scores yet, but there is [a large collection of example scripts/scores here](https://github.com/DigiScore/neoscore/tree/main/examples). Each of the examples there focuses on a specific area of functionality, but there is [an example implementation of a Morton Feldman score here tying many concepts together](https://github.com/DigiScore/neoscore/tree/main/examples/feldman_projection_2), truncated to the first page and a half for copyright reasons.
